diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/ASTNodeUtilsTest.kt

```diff
@@ -62,51 +73,516 @@ class ASTNodeUtilsTest {
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        var isVal = false
+        applyToCode(code) { node ->
+            if (node.isValProperty())
+                isVal = true
+        }
+        Assert.assertTrue(isVal)
+    }
+
+    @Test
+    fun `test isConst`(){
+        val code = """
+            class Test() {
+
+                const val SPEED = 10
+
+                /**
+                * test method
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        var isConst = false
+        applyToCode(code) { node ->
+            if (node.isConst())
+                isConst = true
+        }
+        Assert.assertTrue(isConst)
+    }
+
+    @Test
+    fun `test isVarProperty`(){
+        val code = """
+            class Test() {
+
+                private var name: String? = null
+
+                /**
+                * test method
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        var isVar = false
+        applyToCode(code) { node ->
+            if (node.isVarProperty())
+                isVar = true
+        }
+        Assert.assertTrue(isVar)
+    }
+
+    @Test
+    fun `test getAllLLeafsWithSpecificType`(){
+        val code = """
+            class Test() {
+                /**
+                * test method
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        val list = mutableListOf<ASTNode>()
+        val leafWithTypeList = mutableListOf<ASTNode>()
+        var firstNode: ASTNode? = null
+        applyToCode(code){node ->
+            if(firstNode == null)
+                firstNode = node
+            if (node.isLeaf() && node.elementType == WHITE_SPACE) {
+                leafWithTypeList.add(node)
+            }
+        }
+        firstNode?.getAllLLeafsWithSpecificType(WHITE_SPACE, list)
+        Assert.assertEquals(list, leafWithTypeList)
+    }
+
+    @Test
+    fun `test findLeafWithSpecificType`(){
+        val code = """
+            class Test() {
+                /**
+                * test method
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        var firstNode: ASTNode? = null
+        var resultNode: ASTNode? = null
+        applyToCode(code){node ->
+            if(firstNode == null)
+                firstNode = node
+            if (resultNode == null && node.elementType == CLASS_BODY) {
+                resultNode = node
+            }
+        }
+        firstNode = firstNode?.findLeafWithSpecificType(CLASS_BODY)
+        Assert.assertEquals(resultNode?.text, firstNode?.text)
```

Comment:
```suggestion
        Assert.assertEquals(resultNode!!.text, firstNode!!.text)
```
These variables should be not null by this point, and if they are assert will be green but useless.